docs/_guide/08-tools.md

```diff
@@ -52,11 +52,11 @@ There are a number of IDE plugins that may be useful when developing with lit-ht
 
 VSCode plugin
 
-* https://marketplace.visualstudio.com/items?itemName=bierner.lit-html
+* [https://marketplace.visualstudio.com/items?itemName=runem.lit-plugin](https://marketplace.visualstudio.com/items?itemName=runem.lit-plugin)
```

Comment:
Can you add a short description of what features this has?

Also, I think it should be "VS Code" above, with a space.